prisma/schema.prisma

```diff
@@ -38,7 +38,7 @@ model Profile {
 }
 
 model GitHubDetails {
-  id       String @id @default(uuid())
+  picture  String
   username String
```

Comment:
**thought:** We could add `@unique` to the `username`, because it will always be unique (it is on Github)